src/Tickets/Emails/Email_Abstract.php

```diff
@@ -311,6 +310,24 @@ public function get_attachments( $attachments = [] ): array {
 		return $attachments;
 	}
 
+	/**
+	 * Set email placeholders.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $placeholders the placeholders to set.
+	 *
+	 * @return string
+	 */
+	public function set_placeholders( array $placeholders = [] ): array {
+		$this->placeholders = array_merge(
+			$placeholders,
+			$this->get_placeholders()
+		);
+
+		return $this->placeholders;
```

Comment:
Do you think we should have a filter here so it can be modified elsewhere?

Reply:
Thanks for taking a look! There's already the filter on the "getter" [here](https://github.com/the-events-calendar/event-tickets/blob/9a48621bede450b96e703760537802740e300d12/src/Tickets/Emails/Email_Abstract.php#L299-L311):

https://github.com/the-events-calendar/event-tickets/blob/9a48621bede450b96e703760537802740e300d12/src/Tickets/Emails/Email_Abstract.php#L299-L311